extensions/src/main/java/org/pentaho/platform/web/http/api/resources/ActionResource.java

```diff
@@ -88,14 +86,14 @@ public Response invokeAction(
     @QueryParam( ActionUtil.INVOKER_ACTIONID ) String actionId,
     @QueryParam( ActionUtil.INVOKER_ACTIONCLASS ) String actionClass,
     @QueryParam( ActionUtil.INVOKER_ACTIONUSER ) String actionUser,
-    final String actionParams ) {
+    final ActionParams actionParams ) {
```

Comment:
ActionResource.invokeAction signature was changed to ActionResource().invokeAction( String , String , String, ActionParams). This is the whole point of this PR...??